<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AudioSlotWidget</class>
 <widget class="QGroupBox" name="AudioSlotWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>100</width>
    <height>605</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>60</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>100</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="title">
   <string>Slot</string>
  </property>
  <property name="alignment">
   <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QPushButton" name="slotPlayButton">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../gfx_ressource.qrc">
       <normaloff>:/gfx/icons/audio_play.png</normaloff>:/gfx/icons/audio_play.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>36</width>
       <height>36</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="slotStopButton">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../gfx_ressource.qrc">
       <normaloff>:/gfx/icons/audio_stop.png</normaloff>:/gfx/icons/audio_stop.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>36</width>
       <height>36</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="qsynthMeter" name="meterWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="qsynthKnob" name="slotVolumeDial">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qsynthKnob</class>
   <extends>QDial</extends>
   <header>thirdparty/widget/qsynthknob.h</header>
  </customwidget>
  <customwidget>
   <class>qsynthMeter</class>
   <extends>QWidget</extends>
   <header>thirdparty/widget/qsynthmeter.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../gfx_ressource.qrc"/>
 </resources>
 <connections/>
</ui>
